### DISCUSSION TITLE
Welcome to discussions!

### DISCUSSION THREAD

**Discussion by psifertex** [Vector35/binaryninja-api#1565] [General] [answered]
We're excited to be beta-testing a new feature on GitHub that is designed to enable easy Q&A for GH projects. For the Binary Ninja community, our hope is that the discussions area lives somewhere halfway between GH issues and out vibrant slack community.

This post is a short introduction meant to explain the goal of discussions and help guide users with questions to most appropriate place for their needs. 

First, [file an issue](https://github.com/Vector35/binaryninja-api/issues/new) if you have:
- a bug
- a feature request

Next, [start a new discussion](https://github.com/Vector35/binaryninja-api/discussions/new) if you have questions about:
- "How do I do <x>?"
- The UI
- APIs or automation
- Workflow questions

In short, if the answer to your question is potentially beneficial to someone else, we'd encourage you to use discussions as opposed to slack which is more ephemeral. 

Finally, we still encourage users to [join our slack](https://slackin-dnstndteni.now.sh/) if you want:

- to ask general reverse engineering questions
- have specific customer support questions of the BN team (or use one of our other [many other methods](https://binary.ninja/support/)!)
- to hang out with a cool group of people!

**Comment by psifertex** [comment, marked as answer]
This discussion doesn't require an answer!
